core/src/com/cloud/network/resource/TrafficSentinelResource.java

```diff
@@ -228,6 +239,7 @@ private DirectNetworkUsageAnswer getPublicIpBytesSentAndReceived(DirectNetworkUs
                         answer.put(publicIp, bytesSentAndReceived);
                     }
                 }
+                os.close();
```

Comment:
The input and output streams may not get closed in case of exception, it is better to do any cleanup in the finally block.